messenger.rst

```diff
@@ -860,7 +866,13 @@ The ``amqp`` transport configuration looks like this:
     # .env
     MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
 
+    # or use the AMQPS protocol
+    MESSENGER_TRANSPORT_DSN=amqps://guest:guest@localhost/%2f/messages
+
+
 To use Symfony's built-in AMQP transport, you need the AMQP PHP extension.
+If you want to use TLS/SSL encrypted AMQP you must provide a CA certificate. You should set it through ``amqp.cacert = /etc/ssl/certs`` (path depends on your system) in your ``php.ini`` file or by setting the ``cacert`` setting (e.g ``amqps://localhost?cacert=/etc/ssl/certs/``)
+Using TLS/SSL encrypted AMQP DSN will use port 5671. You can overwrite this behavior by setting the ``port`` setting (e.g. ``amqps://localhost?cacert=/etc/ssl/certs/&port=12345``).
```

Comment:
I assume this isn't a Symfony only setting, and AMQP always uses port 5671 by default?
```suggestion
By default TLS/SSL encrypted AMQP uses port 5671. You can overwrite this behavior by setting the ``port`` parameter (e.g. ``amqps://localhost?cacert=/etc/ssl/certs/&port=12345``).
```